src/containers/notification/store/sagas.ts

```diff
@@ -460,7 +479,17 @@ export function* getNotifications(isFirstFetch: boolean) {
         playlistUpdates: number[]
       } = notificationsResponse
 
+      const existingUpdates = yield select(getPlaylistUpdates)
```

Comment:
nbd, but maybe this block can be extracted to a recordPlaylistUpdatesAnalytics func since it's the same exact block as in fetchNotifications too